src/main/java/org/prebid/server/bidder/pubnative/PubnativeBidder.java

```diff
@@ -137,22 +162,22 @@ private static BidRequest modifyRequest(BidRequest bidRequest, Imp imp) {
     private static List<BidderBid> extractBids(BidResponse bidResponse, BidRequest bidRequest) {
         return bidResponse == null || CollectionUtils.isEmpty(bidResponse.getSeatbid())
                 ? Collections.emptyList()
-                : bidsFromResponse(bidResponse.getSeatbid(), bidRequest.getImp(), bidResponse.getCur());
+                : bidsFromResponse(bidResponse.getSeatbid(), bidRequest.getImp());
     }
 
-    private static List<BidderBid> bidsFromResponse(List<SeatBid> seatbid, List<Imp> imps, String currency) {
+    private static List<BidderBid> bidsFromResponse(List<SeatBid> seatbid, List<Imp> imps) {
         return seatbid.stream()
                 .filter(Objects::nonNull)
                 .map(SeatBid::getBid)
                 .filter(Objects::nonNull)
                 .flatMap(Collection::stream)
-                .map(bid -> createBidderBid(imps, currency, bid))
+                .map(bid -> createBidderBid(imps, bid))
                 .collect(Collectors.toList());
     }
 
-    private static BidderBid createBidderBid(List<Imp> imps, String currency, Bid bid) {
+    private static BidderBid createBidderBid(List<Imp> imps, Bid bid) {
         final Imp imp = findImpById(bid.getImpid(), imps);
-        return BidderBid.of(updateBidWithSize(bid, imp), resolveBidType(imp), currency);
+        return BidderBid.of(updateBidWithSize(bid, imp), resolveBidType(imp), CUR_USD);
```

Comment:
We can avoid bid response and just left currency as is.
`NULL` value will be treated to `USD` by default in PBS-Core.